src/addrman.h

```diff
@@ -332,7 +339,12 @@ friend class CAddrManTest;
         OverrideStream<Stream> s(&s_, s_.GetType(), s_.GetVersion() | ADDRV2_FORMAT);
 
         s << static_cast<uint8_t>(Format::V3_BIP155);
-        s << ((unsigned char)32);
+
+        // Increment `lowest_compatible` iff a newly introduced format is incompatible with
```

Comment:
I think this requires better documentation. At the very least, the `* Serialized format.` comment above needs updating. You might also add `asmap_version` while you're doing that, since the documentation wasn't updated correctly last time either.

I'd suggest something like:

"The lowest version of software that can read this peers.dat file. Update this to be the same as `FILE_FORMAT` when making a change to the file format that is not forwards compatible."

Reply:
I updated the `* Serialized format` comment.